tests/test_evaluate.py

```
@@ -0,0 +1,34 @@
+        return np.mean(y_true == y_pred)
+    evaluator = Evaluator(metric_function=custom_metric)
+    assert evaluator.metric_function == custom_metric
+
+
+def test_evaluator_evaluate():
+    evaluator = Evaluator()
+    y_true = [0, 1, 1, 0]
+    y_pred = [0.1, 0.9, 0.8, 0.2]
+    score = evaluator.evaluate(y_true, y_pred)
+    expected_score = roc_auc_score(y_true, y_pred)
+    assert score == expected_score
```

Ruff

No newline at end of file

Add trailing newline

(W292)